src/app/calculator/waste-water/state-point-analysis/state-point-analysis-form.service.ts

```diff
@@ -43,6 +47,8 @@ export class StatePointAnalysisFormService {
       sviParameter: [sviParameter, [Validators.required, Validators.min(0)]],
       numberOfClarifiers: [numberOfClarifiers, [Validators.required, Validators.min(0)]],
       areaOfClarifier: [areaOfClarifier, [Validators.required, Validators.min(0)]],
+      diameter: [diameter, [Validators.required, Validators.min(0)]],
```

Comment:
Errors on reset button. The 2 fields need to be added to the builder for the empty form.